### ISSUE TITLE
Two questions

### ISSUE THREAD

**Issue by wojtekxtx** [vshymanskyy/StandWithUkraine#181]
1. Why jetbrains is in allow list? Since war started JB has been actively supporting one of major contributor to php ( nikita popov ) who is russian. They also hired him as a developer.
2. Why php is NOT in avoid list? PHP still has popov as well as other russians as their team.

**Comment by wojtekxtx** [comment]
> Also, he's living in Berlin, Germany.

@jdreesen It does not matter one bit. The fact he lives outside of Russia doesnt change the fact he is still russian

**Comment by jdreesen** [comment]
Do you have any evidence he's in favor or supporting the Russian crimes?

**Comment by jdreesen** [comment]
I just wanted to give some context.

**Comment by ink-splatters** [comment]
Hello there from passer-by. 

Disclaimer: I might seem harsh but I declare no intention to deliberately engage anyone in emotional conversation or hate speech here, and I’m sorry if it’s gonna be the case, regardless ofmy intentions.

I don’t know Nikita personally, but on the other hand - I do know many  nice JetBrains guys, some directly, some maybe in 1 handshake. I witnessed ones went public announcing clearly and unambiguously their pro - Ukrainian position.

I live in European country and I personally took part in humanitarian aid contribution sent to Polish border one day. But that’s literally nothing in terms of the value I brought, one might even laugh at it. 

It’s good however that there were other guys who made much more and invested their own cash into help on Polish border, and / or made a series of road trips there.

There were Russian guys among them, well, I would call of Russian origin instead.

It’s also quite traditional in local expat community that one is judged by their deeds.

Through a set of clicks I found myself there and believe me: I have things to do rather than writing this, but seems I got quite weaned from such level of nonsense.

You might want to ban me - please feel free doing that, as well as my opinion might be nothing to you - that’s fine, but at least you are aware:

> “does not matter one bit. The fact > he lives outside of Russia doesnt > change the fact he is still
> russian

that’s an utter _dick stance_ of yours, literally.

The most saddening that it doesn’t make good to literally anyone except maybe one’s receiving ACK from their internal Ego SoC.

It makes much more sense to shoot Russians invaded your territory, even if they did so to surrender in the end as an opportunity to escape Putler’s forces. 

I believe ВСУ doesn’t ( and should not - strictly speaking ) have a capacity to precisely distinguish between “good” Russians and “bad” Russians. That’s a war, сорян (as one may wish saying). If they invaded, they are to be ones least protected.

But you are hopefully not in the situation to pull the trigger, given Nikita would be now crossing the border.

You, on the other hand, do have a chance to figure out if someone’s an enemy to you or a least neutral person; ( worth nothing to say that in some cases you might see a supporter who’s  nothing to do with Russia, in terms of mentality as well ).

I find it valid point about Russian citizens even formally, even if cut all ties to Russia, still being carriers of responsibility for the actions of the _state_.

Because there is a _reason_ and _logic_ here. 

I’m fine with _a_ logic and _a_ reasoning even if I wouldn’t fully agree with the subject. it’s a _lingua franca_, if you want, at least, for mentally capable instances of Homo Sapiens, I hope.

Without intention to trigger you I sincerely, honestly struggle to find any common sense behind your wording.

The only excuse I could come up with is: direct war impact to your family or close friends, and you experiencing psychological trauma here and now.

**I explicitly state if previous sentence is true, I’m to withdraw everything written here and to bring my deep and sincere apologies**. 

Otherwise - dot. 


> > Also, he's living in Berlin, Germany.
> 
> @jdreesen It does not matter one bit. The fact he lives outside of Russia doesnt change the fact he is still russian

**Comment by wojtekxtx** [comment]
@ink-splatters as of russians: its not easy, although highly reasonable, to think your way. But, this kind of mentality russians have is deeply rooted within themselves. This + what they are doing to |Ukraine, makes it hard to show them any mercy. And I firmly bieleve that, if they were allowed to continue this, they would stop only after conquerring whole world.

**Comment by ink-splatters** [comment]
@wojtekxtx 

- JetBrains is not at the list of boycotted companies because they have been clear enough about where they [stand](https://blog.jetbrains.com/blog/2022/03/11/jetbrains-statement-on-ukraine/). The authors of this repo (at least the owner as I can judge) has clear and reasonable position which anyone capable of comprehending.

- sometimes common sense doesn't work, it's sort of norm, and it can happen to pretty much everyone and basically is an emotional response.

IMHO your "mentality" thesis is such that one can _neither oppose it nor defend_, in _argumentative_ way.

Which effectively means that discussion is reducible to sets of monologues instead: you are trying to define the reality in some terms which are _not verifiable_, notwithstanding the fact that: if we agreed on both of us finding and putting here some YouTube links, which depict "Russian mentality" I bet: we would have cross-agreed on the results of each other, afterwards.

The problem is: while we both understand what you mean, it's something out of reasonable discussion' space. When so happens, IMHO it starts unilaterally benefit just one given side of conversation. In simple words, that's called accusation.

Not making the laugh of serious and tragic topic, I still find it acceptable to continue absurd line of thought, as per written above.

So, could you define a Haskel typeset depicting mentality whatsoever and derive Russian mentality from it? 

Some sort of (partial) ordering? Among distinct types or within a type, between the individual "carriers"?

Would be there

```
F: M -> M'
```
where `M` and `M'` are of distinct types? Or is it something hardcoded at Russian "ROM"? and "Mentality" can be fixed by seizing to exist, along with the subject?

What about Jews allegedly had known to have mentality issues, as per Adolf point of view, back to some 1930s? :)

How would you handle Russians, fighting at ВСУ, shoulder to shoulder with Ukrainians? You know there are such, as much as I know. Could their Russian mentality be vetted or they are subject to quarantine? Does it stop in case of death?

Could you define identity mentality ? :)

--

I also know more than reasonably digestible, about what happened in Kyiv oblast' in early days of war. While this incomparable to the feelings of direct witnesses or even to what Ukrainians generally experience, I've been in contact with some having fled Ukraine _literally_ under shelling, to relocate to European city where they continued offline study (with the professors from remote courses, those also helped to obtain residence). 

Those are awesome guys and gals. I believe I have got known them enough to write this. They are also being accused by Ukrainians, by the way, as refused to fight and having fled for study instead (different topic. My IMO is none is ethically eligible to call them not "true" patriots of their country).

There are also mates and colleagues from Ukraine (Kyiv, Kharkiv) there are ones lost their relatives and/or friends; the perished ones are basically in one handshake distance.

I don't diminish the mentally un-processable, horrible, terrifying crimes the carriers of "Russian Mentality" have committed. As well as I consider some _miss. dugina's types_, architects of "Russian world" and related, to be equally responsible for all of this. Every Russian - I think so, but I believe responsibility differs.

Ultimately shame is an indicator of it and it's been always there with me personally, you cannot escape it by the fact you've been abroad, for a while. No one, among people conscious of it, needs to be remembered, it just calls from inside. Some do more about it than others.

While I strictly and ultimately separate myself being related to mentioned mentality, whatsoever, I cannot ask my mom to get me unborn, in order to just remake a delivery at some other geo-location. Well, technically I can, but I estimate the chances of reaching my imaginable goal as relatively modest. That's something which I believe justifies my logic and luckily, as apparently being a common sense, it finds wide support among my network, also officially. As it also apparently seems to be aligned with at least what's called human rights, at least as how it's defined as per EU regulations / manifests

**Comment by wojtekxtx** [comment]
@ink-splatters many valid points, but let me clarify my anti-russian stance.

No, this war had not traumatised me personally; well it traumatised me, but the same way it traumatised every single, clearly-thinking, human being.

Why Im against jetbrains? Simply for their support of php, and php-foundation ( they [spent $100k](https://opencollective.com/phpfoundation) on it) knowing that [first and second place in contributors list to the php/php-src is taken by russians](https://github.com/php/php-src/graphs/contributors).
As for your linked JB stand: sorry, but this is nothing more than marketing BS. They clearly showed it by their money transfer to php-foundation opencollective. They all are bunch of hypocrites! Thats it!

Why Im against `php`? For not taking actions against their top contributors ( sergey and nikita ). Thats simply it.

**Comment by ink-splatters** [comment]
> @ink-splatters many valid points, but let me clarify my anti-russian stance.
> 
> No, this war had not traumatised me personally; well it traumatised me, but the same way it traumatised every single, clearly-thinking, human being.
> 
> Why Im against jetbrains? Simply for their support of php, and php-foundation ( they [spent $100k](https://opencollective.com/phpfoundation) on it) knowing that [first and second place in contributors list to the php/php-src is taken by russians](https://github.com/php/php-src/graphs/contributors). As for your linked JB stand: sorry, but this is nothing but marketing BS. They clearly showed it by their money transfer to php-foundation opencollective. They all are bunch of hypocrites! Thats it! From one side they said [this](https://twitter.com/jetbrains/status/1496786254494670851) and [this](https://blog.jetbrains.com/blog/2022/03/11/jetbrains-statement-on-ukraine), but from the other side they:
> 
> * showed full support for nikita ( by hiring him ),
> * showed full support for php-foundation ( by donating $100k ).
> * showed that they will relocate its russian employees instead of cutting ties with them. (most of their coredevs are russiansw but they are based in Czech Republic; solely to circumvent sanctions).
> 
> Just to sum up: **JB is russian company** and as such should be:
> 
> * banned from doing business worldwide,
> * held accountable.
> 
> Why Im against `php`? For not taking actions against their top contributors ( sergey and nikita ). Thats simply it.

Ok, thanks a lot, this is exhaustive and far more clear. By some reason I missed this point about php…

**Comment by wojtekxtx** [comment]
@ink-splatters whats your course of action now?

**Comment by CarlosCapDev14** [comment]
Guys why I'm suscribe to this conversation? 😳

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by wojtekxtx** [comment]
@schernichkin Im blaming not him but **the rest of `php core team` and JetBrains** for defending him. Fact that he is Russian is enough for (sadly almost) everyone to cut ties with him.

**Comment by schernichkin** [comment]
Did I understand correctly that you discredit people only for their nationality? Is it your personal position, or do other members of the community who placed the banner share it?

**Comment by wojtekxtx** [comment]
@schernichkin 

> Did I understand correctly

Yes, correct. But you see: I dont normally do this; this is first time I do this. Nikita is Russian and all Russians are the same. This mentality runs deep in them. Thats one reason why there are sanctions against Russia in place.

>  Is it your personal position,

Both personal, and shared amongst others...........

> do other members of the community who placed the banner share it?

no idea.......

**Comment by schernichkin** [comment]
>Nikita is Russian and all Russians are the same. This mentality runs deep in them. `

I see. But how do you determine who is Russian and who is not? By ancestry? What if one of the parents is Russian and the other, lets say, Ukrainian? Perhaps some sort of chart should be used for this purpose, similar to this one: ![](https://upload.wikimedia.org/wikipedia/commons/c/c7/Nuremberg_laws_Racial_Chart.jpg)

**Comment by wojtekxtx** [comment]
@schernichkin 
> But how do you 

I dont. Its enough to have Russian name&surname.........

> What if one of the parents is Russian and the other, lets say, Ukrainian

That will give us half russian, still. So, still qualifies as Russian, sorry.

**Comment by ink-splatters** [comment]
@wojtekxtx @schernichkin 

Mein Respekt, mein Furher: auch mit diesen verdammten Juden in den 1940er Jahren… Obwohl unsere Öfen rund um die Uhr lief, haben wir unsere Kapazitäten immer respektiert und konnten sie NICHT alle auf einmal verbrennen!!

Wir können es auch nicht mit verdammten Russen machen!

Daher die Karte.

_Das **Ordnung**_, das brauchen wir!_

Dumme Kinder nennen es neus SCRUM und Agile.
Bestellen Sie mich einfach und bauen Sie auch den Überofenofen für sie. Bis wir es nicht haben, lass uns bodenständig sein!

Diese SS Offiziere… mit den Hinrichtungen vom Zweiten Weltkrieg gegangen waren. _Sie schlossen sich schließlich den Juden an…_

Ich bin sicher, wir hatten es richtig gemacht, mein Furher…

Hat alles in Ordnung gemacht!!!…

**Comment by wojtekxtx** [comment]
@ink-splatters what the hell?

**Comment by wojtekxtx** [comment]
@ink-splatters 

> their pro - Ukrainian position.

You mean the one by JetBrains. **Its bullshit.** They prove it by their actions aferwards:
* giving $100k in donation to php foundation,
* supporting Nikita
But its quite normal for them, to support their co-native.

The fact that you are arguing here in avour of Russians means either:
* you understand nothing so you condemn nothing,
* you (not literally) say that sanctions against RF are baseless,
* you are literally monster without even slightest sense of morality, disgust, etc

**Comment by jdreesen** [comment]
How do they support Nikita? He is no longer a JetBrains employee and is not part of the PHP Foundation.

**Comment by wojtekxtx** [comment]
@jdreesen 

> He is no longer a JetBrains employee and is not part of the PHP Foundation.

Both statements are true. However, he was fulltime member of PHPF when money was transferred from JetBrains. Only after fuss was made they (JetBrains) released these beforementioned statements.

**Comment by ink-splatters** [comment]
> @ink-splatters what the hell?

> @ink-splatters what the hell?

The PHP community is linked to financing Russia which drops a shadow on JetBrains still participating in money flows supporting the war - is logical point. 

You put the proofs, I followed them and appreciated your logical approach.

You asked me what I was supposed to do. I could not allocate time for this thread so answering now:

practically - nothing, I was evaluating JetBrains CLion and GoLand which are great generally, I might’ve used them for free, but my way of development doesn’t really justify installing the whole gigabytes of Java shit, while I can live happy without it; plus all the smart features are not that smart in fact.

Thus I landed on nvim + tree sitter and language servers for what I need. This is deployed using nix-darwin and so far I’m quite happy having mostly concise and powerful environment.

I’d first speak with JetBrains employee(s) in person, asking how it feels to know your Employer sponsors the war, before potential readiness to use my social media to spread this information, double checking all the evidence.

—-
Now let’s talk about very bad part.

I admit that my previous post was a fat one, but if you don’t see what the problem is, I can try being even more obvious, in order for you to realize it:

_your clear racist views can’t be excused by the fact that your country is a victim of horrible Russian aggression_

Your feelings about horrible (war crimes) committed in your country , also against women and children are hard to process, for one haven’t been a witness;  and I don’t doubt for a second about their truthfulness and understand the level of hate you have towards those monsters of Russian origin who have been doing it. It all goes way beyond manageable.

But apparently you are 18+ person and therefore you are fully responsible for your actions and expression of your feelings. Though they are really understandable, they cannot justify your unleashing hate speech, humiliating the whole nation, just because you are crossing the territory of someone else freedoms.

And it’s COMPLETELY nothing to do neither with Ilya, nor with PHP community, nor with JetBrains.

This is just your behavior, just because you chose to behave so. The concrete real problem is that no single Russian can escape your judgement - it makes your speech IMO hardly distinguishable from far right nationalists of both modern and WWII times.

Unless you commit crimes you are welcome to continue your narrative, but keep in mind the trade-off between this and how far you distance yourself from healthy inclusive communities and from common sense whatsoever. If you care of course.
Dot.

**Comment by ink-splatters** [comment]
>this is first time I do this. Nikita is Russian and all Russians are the same. This mentality runs deep in them. **Thats one reason why there are sanctions against Russia in place**.
> 

Maybe here lies the problem as this is **ultimately misleading point**.

In reality the sanctions are for impacting the individuals directly accountable for war crimes and to throttle state’s _economy_ to (ideally) undermine the capabilities of continuing the war, also affecting everyone as everyone is tax payer.

The sanctions (officially) are nothing to do with someone’s mentality, being undefinable abstract subjective construct: everyone understands it their own way.   

A person having worked and paid taxes in Russia who decided to flee Russia or even more - had done so _years ago_; also who avoids any financial indirect contribution (direct is not even a topic) to the criminal regime, acts in accordance with their anti-Putin’s Russia position, by doing so should escape the common attitude one had towards a National of terrorist state.

No Russian can escape your attitude, regardless of views and actions,  that’s what myself and other participants have been trying to say there.

**Comment by wojtekxtx** [comment]
@ink-splatters Honestly Ive no idea why/on what basis) you make assumption that Putin invaded our country. No, Putin **did not invade Poland.** You have not read my profile even.

Let me ask you this: are you either
* Russian/of Russia origin, or
* being paid for defending Russians?

Serious question. By tone of our reply, I can clearly see this, but I just dont know which "dot" it is.

Either way: its clear that you have different views than myself. It all ok. I respect that. Am not going to argue that there is only one way of thinking; no; everyone is entitled to their beliefs.
But: when (almost) every civilized country sets sanctions against RF, than there is clearly something not right; besides these sanctions have to based on something serious. Goverments dont usually issue sanctions based on someone doesnt like another country's leader (just example).

> But apparently you are 18+ person and therefore you are fully responsible for your actions and expression of your feelings. Though they are really understandable, they cannot justify your unleashing **hate speech**, humiliating the whole nation, just because you are crossing the territory of someone else freedoms.

You call generally known FACTS a hate speech? Seriously @ink-splatters ? Yes, Im well over 18, I am fully responsible for my actions; correct. But what you clearly do not understand that facts dont have to be nice/ly sounding, speaking facts outloud is not spreading hate speech. Although negative facts can ( and in some communities will ) lead to hate speech.

**Comment by jdreesen** [comment]
> This is the viewpoint shared by 99% people on this planet.

You rather mean: "This is the viewpoint shared by 99% of people in my bubble."

**Comment by wojtekxtx** [comment]
> > This is the viewpoint shared by 99% people on this planet.
> 
> You rather mean: "This is the viewpoint shared by 99% of people in my bubble."

@jdreesen If by buuble you mean our planet, than yes, this is what I mean 👍

**Comment by ink-splatters** [comment]
> @ink-splatters Honestly Ive no idea why/on what basis) you make assumption that Putin invaded our country. No, Putin **did not invade Poland.** You have not read my profile even.

Well I have, but I missed your origin indeed. Now, having learned it makes me think that you probably sadly understand better than me what the race-based discrimination is, and to which scale of genocide it had led to, being elevated to absolute in 1930-1940s.

> 
> Let me ask you this: are you either
> 
> * Russian/of Russia origin, or
> * being paid for defending Russians?
> 
> Serious question. By tone of our reply, I can clearly see this, but I just dont know which "dot" it is.
> 

Well, I was born in Siberia. To my best knowledge it’s still 

<details>
  <summary> Russian territory. </summary>
  

Apart from okeish to shitty industrial cities and some seriously good scientific institutions known to conduct joint high-profile studies with labs worldwide (physics especially, that was de-facto killed by Putler’s invasion):

you can find Pacific North-West’s beauty there, think Cascade mountains / mt. Rainier‘s kind; but it doesn’t lend itself to car fans. And trails to the gems of it don’t necessarily exist, u’d need to create yourself sometimes.

Good that Putin shall perish soon but the beauty remains. As well as it had existed long before it was called Russia.

</details>


> Either way: it’s clear that you have different views than myself. It all ok. I respect that. Am not going to argue that there is only one way of thinking; no; everyone is entitled to their beliefs. But: when (almost) every civilized country sets sanctions against each other, then there is clearly something not right; besides these sanctions have to based on something serious. Goverments dont usually issue sanctions based on someone doesnt like another country's leader (just example).
> 
> > But apparently you are 18+ person and therefore you are fully responsible for your actions and expression of your feelings. Though they are really understandable, they cannot justify your unleashing **hate speech**, humiliating the whole nation, just because you are crossing the territory of someone else freedoms.
> 
> You call generally known FACTS a hate speech? Seriously @ink-splatters ? Yes, Im well over 18, I am fully responsible for my actions; correct. But what you clearly do not understand that facts dont have to be nice/ly sounding, speaking facts outloud is not spreading hate speech. Although negative facts can ( and in some communities will ) lead to hate speech.

You are trying to explain me the obvious things like I don’t understand it. Well you could treat me a bit less stupid, it’s of a potential to save some time for both of us.

But, what is really hilarious is that I indeed _support_ your thoughts in many regards, and I was trying to make it clear as well, somewhere above.

So it’s not even applicable to discuss everyone’s different line of thoughts, but rather common, except this one very annoying fact I emphasized above: 

**> > No Russian can escape your attitude, regardless of views and actions**

That’s what immediately shadow-mounts itself on top of your, otherwise, logical and reasonable thinking, rendered verbally in this thread.

As ordinary, _non-Boston-Dynamics-originating Homo Sapiens_, of flesh and blood (and maybe also the thing behind ChatGPT, lol?) is built this way that they are (myself in this case) prune to rather harsh emotional reaction triggered by some substantial (verbal) assault. 

When doing my best reacting on verbal assault, I try doing some pre-flight reality check, before actually getting mad about it: like  making sure that I did not act like a jerk and got what I had opted for :)

This time I seriously have put the whole test suite here but stil struggle to swallow rather evident shitball in my borsch :)

it’s not about my feelings towards the Cooker, per-se, but just about my reaction, which I fail to find disproportionate, still I hate to react like this

**Comment by ink-splatters** [comment]
As analogy to _gender pronouns_ I feel the urge for _origin pronouns_.

Call me Siberian or f*cking snowman if you want :) 

I’m one infinitesimal away from putting it to my profile actually :)

**Comment by wojtekxtx** [comment]
> I indeed support your thoughts in many regards (...)

@ink-splatters you changed sites rapidly. Wow. You have tried to convince me Im wrong and now, after seeing that you WILL NOT have success, you claim to understand me. Wow.

> Well, I was born in Siberia.

So, you ARE Russian; I was correct.

**Comment by ink-splatters** [comment]
> @ink-splatters you changed sites rapidly. Wow. You have tried to convince me Im wrong and now, after seeing that you WILL NOT have success, you claim to understand me. Wow.

> > struggle to swallow rather evident shitball in my borsch :)
> 
> Well deserved shitball in fact.

Do quote my “change” that you see.

I’m sure I’ve been consistent in that I condemn Russian aggression, everyone having compassion or indifference regarding it.
And it’s worth nothing to say I condemn sponsoring it in any way.

You provided argumentation regarding JetBrains -> PHP link, - I noted it, as: if by chance the PHP’s Russian governance also means paying taxes in Russia or contributing to Russian militaries’ cash flows, by any other means, that’s a bold one and I would not be ignoring it.

I’m not part of PHP community and not affiliated with JetBrains, I don’t use any products of those, therefore I don’t feel immediate public responsibility and I’m using this to take a better look into it first, as these sort of charges against people or communities I take seriously and if proven by facts, that’s a reason to take actions. But just because it’s no joke shit, one would better off being really sure about their evidence first.

Because _they are Russians_ is not an evidence at all, **but just a racist rant**. And if strip racism from it, it’s infant level argumentation. As I come from the point that you are not stupid either, you just can’t misunderstand what I mean by that.

Sadly, however, that your reasoning drops at the very first occurrence of

`[Rr]ussian[\w]+` 

in it, turning into something you would claim to make sense, but in fact is centric to your hating Russians and therefore one would better kill themselves against the wall than ever try to argue with you regarding this, because as far as you stick to your current views, a conversation lacks any _space_ for argumentation, as no argumentation can defend your ultimate “because fuck Russians that’s why” point.

> 
> > Well, I was born in Siberia.
> 
> So, you ARE Russian; I was correct.

Looks like you used caps to emphasise the importance of it like “so that NOW it all becomes clear to me, just as I thought”. Lol.

Rest assured I’m doing just fine man, and my conscience is crystal clear in this regard, as I’ve NOT been paying taxes in Russia for quite a while, for some 5+ years already. And I’m unambiguous in my position and actions, also beyond internets.

And thought Russian origin is definitely nothing to be proud about, nowadays, it’s a nonexistent problem for me if compared to your problems like ultimately ill-formed attitude.


> 
> > struggle to swallow rather evident shitball in my borsch :)
> 
> Well deserved shitball in fact.

Can you be so kind to elaborate it for me? Please💕

I admit being not too much polite but mehh, it would be very dull reason to have deserved the aforementioned round object in my immediate vicinity (borsch I mean), isn’t it? Or I’m wrong?


```
// Hint: please tell me I deserve it because I’m Russian!

```
Now the serious one: you tell me, **what _exactly_ have you done for supporting Ukraine?**

I mean, c’mon, open up! As you requested it from me.

**Comment by wojtekxtx** [comment]
>  what exactly have you done for supporting Ukraine?

@ink-splatters here you go:
* Ive (personally) organised transport of pets (dogs, cats, other home animals) out of Ukraine. The day war began. Adopted 2 of them: cat&dog; both lovely little innocent creatures,
* Ive helped in housing of one family of 4. For free. Meaning Im covering all costs of living here; I paid for all the formalities etc.

Dont get me wrong: ou personally may be nice guy, but, in the light of your nationality as well as sanctions, one may experience hard time even "speaking" (either in person, or via electronic means) with you. Thats just yet another fact.

**Comment by cristaloleg** [comment]
Thread reminds me https://pbs.twimg.com/media/FQyyvrnWQAERaRY.jpg:large

**Comment by wojtekxtx** [comment]
@cristaloleg exactly this man :)

**Comment by ink-splatters** [comment]
> > what exactly have you done for supporting Ukraine?
> 
> @ink-splatters here you go:
> 
> * Ive (personally) organised transport of pets (dogs, cats, other home animals) out of Ukraine. The day war began. Adopted 2 of them: cat&dog; both lovely little innocent creatures,
> * Ive helped in housing of one family of 4. For free. Meaning Im covering all costs of living here; I paid for all the formalities etc.

Thanks for answering. Well, my family's contribution to Amsterdammers of UA origin, and another one which was sent to Polish border seems to be way less. My respect to you.


> Dont get me wrong: ou personally may be nice guy, but, in the light of your nationality as well as sanctions, one may experience hard time even "speaking" (either in person, or via electronic means) with you. Thats just yet another fact.
> 

That's correct and that's what rather distant 2-handshakes-away acquaintance experienced being unable to attend planned interview with UK's office of some company. _Zoom_ interview - someone was literally forbidden to have a call with a Russian citizen; they eventually did fine, moving the process to Dutch office AFAIK. That sort of smells, but is understandable; furthermore it seems to be some _legislation_ apparently ( AFAIK ). Personally I've been lucky so far not to have issues, working among other, with UK-based freelance customers. But I'm also not Russia-based which is important.

> > I admit being not too much polite
> 
> This + 

well now seems a good moment for me to say: sorry

> 
> > conversation lacks any space for argumentation
> 
> Regarding this (Russians, their invasion, sanctions against RF etc) yes, there is no room for arguments. But simply because there are no words that will (even partly) justify what your nation is doing.

There is no words indeed, it's all completely fucked up, beyond comprehending, beyond processing by any sane mind around. So people respond as per their ability to do so.

I relate myself inevitably and directly to what has been taking place, of course, as it's not Poles, or Slovaks or any other nation; definitely - not Ukrainians; but it's Russians who have engaged into a full scale war _and_ keep on regularly committing war crimes, killing innocent people. These feelings are mainly shame and grief, never need to be remembered about and always come in related situations.

But what made it stronger than anything else for me, is the luck of meeting some decent nice guys, well educated, very reserved (that's I rearly experience). We have a lot in common with. There is a person I actually admire, among them. They studied in Kyiv and they were directly impacted by this war, maybe I mentioned, but they were evacuating under shelling.

What made a huge impression on me is that those concrete people are of very much the same "cultural code" I relate myself to. That's not a sound wording but actual thing: I never ever considered my circles Russian-only per-se, as they were all the way CIS, territory-wise, at minimum, as well as some guys were from Baltic States and US. Besides, quite a lot acquaintances all over the world. Modem internet + bunch of BBS was enough to have it all, lol. People did come from abroad to study in the university I studied in; it didn't look something unusual. Creativity, sharp and open minds were very much honoured. I also happen to love Ukrainian music of mid 90s-2000s which is unique and I NEVER can imagine Russian band making this, it's just different, even if sang in Russian language. Apart from some Ozzy and Metallica which is nothing surprising to listen to, Ukrainian musicians prevailed in my playlists vs Russians actually.

My ex(early 2010s) is Ukrainian girl, she's one of very limited number of people impacted my life drastically. In the very positive way. We sang together in Ukrainian... Well, sadly I don't speak it at all, though comprehend mostly everything, audibly as well.


Now imagine: if my circles - the people I know and respect, are the targets, if someone wants the whole culture seizing to exist; the culture which is not my original, but which has become seriously important part of my life.

Well those doing so are the actual enemies of mine, and I have zero time spent and zero fucks given for any doubts or regrets based on our common nationality; it actually only adds up to my hate, as all this shit had spawned at the territory I had considered my home, and this shit is called Russians obviously.

The same is very well mutual of course, I am potential target if only some right Russian person of views "patriotic" enough would have got bloody hands on me, having figured out my whereabouts. I have zero doubts about the most fucked up end of myself, potentially.

It sounds a bit less bad if I had flown to Pulkovo or Vnukovo/Domodedovo, say tomorrow. Which I can't physically, lol. Apart from taking a detour via Georgia or Armenia. As the "caring dictator" has also cared about Aeroflot's future becoming suddenly inexistent; and no one from EU neither is allowed nor wants, I guess, flying there. So, if I had made it: if only one would have appeared to _want_ it  for a second, I would've been locked not even for those posts, but for some several years old tweet, way more innocent. If you'd think I'm overreacting: the boyfriend of my another ex had been charged long before the invasion happened; though it was very evident nationalist text, today no one needs this efforts, there is a priority pass to FSB major's room for some rendezvous implying them to use offensive security tools against your... khm, backend, saying so. Which is also unlikely "bare metal" like it's the case with quite durable one which Bender from Futurama has.

And you might say it's stupid but I was seriously investigating if I can be caught and repatriated at <details><summary>Russian Embassy</summary>
>Unfortunately there are valid reasons to visit the embassy, which Russian pass holders abroad can't avoid, even if have opted for EU passport journey; and I'm not speaking about regular expired pass update as that's now unwanted option.
>
>Now imagine if some person is to have EU pass soon and needs to visit the aforementioned institution. And becomes caught, lol. I mean it's a challenge to experience that level of bitch at the fortune's chair :) But in the age when grotesques become real I would've been surprised. Luckily, local laws always apply. Not too much obvious :)
</details>

Now - you are the reference example of quite common approach to the problem - nobody actually cares which "kind" of "Russians" who is who. No allegations from my side now, honestly, it was emotional - yes, but I spit it out already. 

The very practical thing: when one has no leverage to oppose themselves against the shit they have gotten into, it's also not realistic for their voice to be heard; and I responsibly claim it's sometimes worth to be.

This:

[Ilya Yashin](https://en.wikipedia.org/wiki/Ilya_Yashin), apart from other puppet "liberal" cuckolds in todays Ru politics spewing nonsense and doing nothing to stop the war (by which constituting a single legion with Putin); has balls of steel, as he has chosen a jail term over fleeing Russia: 8.5 bloody years as it has occurred, just for a social media post about massacres Russians committed in Ukrainian Bucha. For spreading the truth.

if the thread makes it loud enough for people just to learn about heroes like Ilya, it's already very much worth existing and would be very satisfying result of my keyboard monkey work (I broke up with Siri, you know).

I'd also seriously respect my opponents if they would dare to express their position to guys like Ilya, face to face or at least in any other direct way. That's not for engaging into fight but for having an opportunity to personalize the source for their conclusions, making it as direct as possible with practically zero external noise. As I normally believe in humanity, even at the moment, I also believe there is a potential 

_to break the ice one day_. 

It's only that this ice now is 6 feet under the tons of violence, gore, brutal deaths and destroyed lives.

The most fucked up Gindcore album has become a life. What a surprise. Given that decent Grindcore is devoted to politics and war crimes

**Comment by ink-splatters** [comment]
I leave it here in the memory of

**Mieszko Talarczy** † 

who sadly tragically passed away in [Tsunami disaster](https://en.wikipedia.org/wiki/2004_Indian_Ocean_earthquake) in 2004

There is never a bad moment for this masterpiece, but this fucking war made this song frighteningly relevant. This is also for some beasts parasitising via taking away innocent lives, to find their morbid reflection in between the lines.





![](https://cdn.shopify.com/s/files/1/0269/8962/5424/products/Nasum_Helvete_CD_700x.png)


[NASUM - Relics](https://www.youtube.com/watch?v=BaL36_aJBEQ)

```
Take aim at your promised land
Where would you be without it?
Destroy your legacy so grand
No "peace" will halt your vengeance

And so again...
What makes you settle so convinced?
A relic that leaves no room for peace?

So fragile it has always been
Outside main focus

Your focus should be suffering
Or have their cries made you go numb?
Can't you see the enemy within?
Blind with hate you desecrate what brought you here

But then again...
It makes you settle so convinced
A relic that leaves no room for peace

So fragile it has always been
Outside your focus

Extinction - your final goal
Now you're the relic

And so again...
You stand here settled, so convinced
Blindfolded, hatefilled, without regrets

The centre of attention
Doomed, you stand and fight the flames...
Lit by your own hand
You can't float upon the river of flame

**Comment by wojtekxtx** [comment]
@ink-splatters not Talarczy, but Talarczyk. Pleasee see [this post](https://en.wikipedia.org/wiki/Mieszko_Talarczyk)

**Comment by schernichkin** [comment]
>puppet "liberal" cuckolds in todays Ru politics spewing nonsense and doing nothing to stop the war

Это кто, например?

**Comment by wojtekxtx** [comment]
> And you might say it's stupid but I was seriously investigating if I can be caught and repatriated at

@ink-splatters  Thats NOT stupidity; its reality. Each & every person with **any form of ties** with RF should have been caught and repatriated to RF long ago. Any person wanting to leave RF ( =go abroad) should be banned to do so the day the invasion started. Sorry, just my 2p's.

> [Ilya Yashin](https://en.wikipedia.org/wiki/Ilya_Yashin)

@ink-splatters well, he is not a hero at all. He just saw oportunity, took it and was/is "punished" for it. He is just one more victim of RF regime. Thats all...

>  I also believe there is a potential to break the ice one day.

@ink-splatters not possible. At least not in forseable future. If at all......

> My ex(early 2010s) is Ukrainian girl,

@ink-splatters its good she broke up with you. Means that she is perfectly sane. Not sorry at all. You deserved it.

> Now - you are the reference example of quite common approach to the problem - nobody actually cares which "kind" of "Russians" who is who. No allegations from my side now, honestly, it was emotional - yes, but I spit it out already.

@ink-splatters know what? You better just shut up man. Seriously: if I was from nation that started war and committed all these horrible war-crimes, I'd just shut up and take what it is to take (guilt etc).
Very last thing that you, being from RF, should do, is to throw allegations at someone (doesnt matter emotional or not). The fact that you have thrown allegations means you have quite a temper which is typical to people from RF.
And thats exactly what makes me not wanting to exchange messages with you. Just proves what Iv been stating all over: Russians dont change; their mentalliy runs deep.

**Comment by schernichkin** [comment]
>I want to meet no Russians face-to-face, it'd be bad for them. I mean I would not hurt them physically (I would be ashamed to touch them even), but Id tell them real truth about all this.

Hmm... I guess you would be able to have such opportunity. I'm just wondering what you will do with any "russian" if you'l meet one? Except of making outright and, let's say, "questionable" statements that would be ignored by any sane person regardless of his nationality or origin.

**Comment by ink-splatters** [comment]
> @ink-splatters not Talarczy, but Talarczyk. Pleasee see [this post](https://en.wikipedia.org/wiki/Mieszko_Talarczyk). Though I fully agree that this is masterpiece. Seriously.

Aww, that's annoying typo... my bad. Fixed it, but could come up with the response just now.

**Comment by wojtekxtx** [comment]
@schernichkin and look who is here...... (yet) another Russian citizen. What the hell?

**Comment by ink-splatters** [comment]
> > And you might say it's stupid but I was seriously investigating if I can be caught and repatriated at
> 
> @ink-splatters Thats NOT stupidity; its reality. Each & every person with **any form of ties** with RF should have been caught and repatriated to RF long ago. Any person wanting to leave RF ( =go abroad) should be banned to do so the day the invasion started. Sorry, just my 2p's.

Do read again about repatriation: you seem to have missed it: I was talking about being repatriated, in theory, _at Russian Embassy_, by Russians. You obviously mean the member states to take an action and help Russian residents reunite with their motherland. Both result in similar things, but repatriation conducted by Russians I would not wish even to my enemy.

As for being expelled from residency - well, we live in the world of different opinions coexist with each other, normally; furthermore your point seems not absurd at all, but has its logic. You might want to speed-up this process by being elected to local or EU Parliament, following up some efforts, and check your unconditional anti-Russian stance' popularity, at larger sample size, and with the access to legal instruments as the bonus. 

Hope the fact that I aint gonna support your programme would not come shocking to you.

> 
> > [Ilya Yashin](https://en.wikipedia.org/wiki/Ilya_Yashin)
> 
> @ink-splatters well, he is not a hero at all. He just saw oportunity, took it and was/is "punished" for it. He is just one more victim of RF regime. Thats all...

Sorry, what is the opportunity you are talking about?

I can admit that there may be higher bar for being called "hero". And I apparently see that's how you feel.

But it looks from your wording like you are treating Ilya not only _not a hero_ but dismissive whatsoever, like his deed is worth nothing. Or at least not enough to be respected for.

To your opinion he has just exploited some "opportunity" (which one?) and hence benefitted from it (by definition of "opportunity" word). Please correct me if I'm wrong about what you meant, seriously. But that's how I understood it.

"punished" is quoted, why did you quote it? Is the thing he got into, not "true" enough or? Or it's me who sees inexistent things and you meant nothing like this?

My 2c is that opportunities he got use of, are:
- the opportunity _to remain a human_ and
- the opportunity to live according to his moral principles.

Those two are accessible to everyone of us per-se, but do cost very different, depending on the concrete situation, and depending on, of course, different level of peoples' principles.

It's indeed should not be heroic at all - to publicly condemn war crimes (which Ilya did, talking about Bucha), but only if you are happy enough to reside at _democratic_ state. His public visibility paid back with his case becoming known about; it's also made this jail term inevitable for him.

Ilya has made a choice which he believed was the only possible to make. Again, nothing to be necessary proud of, but still there is nuance: in 2022's Russia, _one can be locked for a long, just for condemning war atrocities an making people aware of it_. So to simply follow his believes he had to pay a lot.

The awareness has spread quite widely about his case, making the overall awareness about today's Russia's ugly face, somewhat increase. I see this a benefit for the society and beyond his own gain from it. 

IMO also that it's not Navalny or other suffered from the regime, but Yashin is, who has made a simple move, but very bold one and with the strongest message in it. All this combined with his pure reputation and being simple good person without "double bottom". Even if to consider this a "PR" (which is somewhat dirty wording to me, in regard to Ilya's honour), he is going to pay for this "PR" with 8.5 years of his own life, which I believe is well enough for people to stop doubting about the truthfulness and high morality of his intents.

> 
> > I also believe there is a potential to break the ice one day.
> 
> @ink-splatters not possible. At least not in forseable future. If at all......
> 
> > My ex(early 2010s) is Ukrainian girl,
> 
> @ink-splatters its good she broke up with you. Means that she is perfectly sane. Not sorry at all. You deserved it.

LOL, not only was it some 10 years ago, and either you want it or not, the situation had been different back then.

You draw conclusions regarding my private life (I mean: you decided that she broke up with me, and not myself) which is per-se not to be happy about, but also you do this in a pretty biased way, like it's clear for you that I deserve it.
What's behind this "guess" if not another attempt to insult me? 

This thread has seen a lot and well deserves to avoid witnessing Santa Barbara as some rotten cherry on the fucked up cake, but: I could have shed a light on eventual consistency regarding our break up, lol, and who was the initiator. 
Only it would serve no purpose, as I believe it can't change your opinion and conclusions, can it.

> 
> > Now - you are the reference example of quite common approach to the problem - nobody actually cares which "kind" of "Russians" who is who. No **allegations** from my side now, honestly, it was emotional - yes, but I spit it out already.
> 
> @ink-splatters know what? You better just shut up man, seriously. 

What _exactly_ has made you feel offended? My phrase about your behaviour being "reference example"? Honestly you'd better blame me for anything else, but this is really silly situation because I really didn't mean to offend / hurt you at all, and what I said is actually what I meant: unconditional exclusion of any Russian by their origin is very _common now_, and I really _cannot see_ what exactly is wrong with my attributing this behavior to you:
- thru the _whole_ thread you are emphasizing your point that _all Russians are the same_. 
- This:
>Just proves what Ive been stating all over: Regardless where they live (in RF or abroad) Russians dont change; their mentalliy runs deep.
so you basically finished your message with _exactly what was talking about_.

I mean, really, think about it. And if you don't like my wording - seriously - sorry for that, but what am I wrong about _semantically_?

**Comment by ink-splatters** [comment]
>if I was from nation that started war and committed all these horrible war-crimes, I'd just shut up and take what it is to take (guilt etc). 

But you are not. And let's be honest, you can conduct mental experiments, but you don't belong to a nation in charge of war crimes and I'm happy for you. But "guilt etc" - this "etc" is a sign of your being not too much interested in the details, is not it. That's fine per-se, as honestly why should you care? You shouldn't, but as you tell me what I should do, I feel the urge to bring my exact experience, so if you choose to continue appointing me to do what you think I should do, you'd have the context and hence could judge better.

- I've had enough of my guilt and there is number of feelings which have never left me since the beginning of this war, and maybe they never will. The least you'd need to care about is to remember me about my responsibility as nothing can do better than conscience which I believe I have, does from inside.

- According to the psychology basics, common sense and whatever you would want, and I urge you to ask for the sources (like scientific studies) if you doubt, but otherwise believe me: **_one cannot bear a given responsibility / be accountable for anything if his mental health or mental condition / wellbeing is severely compromised_**. ( I'm saying also: "condition" because e.g. depression is often reversible and can be tightly linked to the situation, and not being necessarily "hardware" (mental health) problem.) 
- In the most of the criminal laws all over the world this is probaby more known as (in)sanity and if a person commits a crime being in a _frenzy_, they cannot be appointed legally responsible for their actions and instead of that are placed in mental ward or discharged whatsoever.

Why I mention this: the guilt.

When it's a natural feeling which serves the purpose of one's "healing", via reflecting about something that needs to be reflected about, it's a normal, it's very humane feeling.

But if it's not stopped, either because a person chooses so, or because or harassment or just the situation concluding this way that _there is no way to pay back_ for one's guilt
and / or process it on the basis that actually one cannot take some limitless amount of shit appointed to them; and maybe they _should take it_ but they _cant_.

It leads to psychological (if not physical) suicide. That's why I strongly disagree with you that people should call for someone's taking all the _guilt_, shut up and hold their face down, as by this one would steal the chance of a person / people in charge, to actually _hold their responsibility_, especially _thru the action_.

Honestly imagine some person drinking the shit of themselves, trying to burry the guilt at the bottom of the bottle: they would have their bitter, miserable end, but _regardless_ of _if_ they deserve this end or not, _it's nothing to do with responsibility_.

This extrapolates to any exposure of oneself to destructing agents as heavy drugs and alcohol, reckless driving or another dangerous behaviour. Committing suicide as the "brightest" of it: all these are just misery aimed to _avoid being responsible_, _avoid
paying back the damage_, _avoid changing oneself in attempt to not meet a shit every morning, when looking in the mirror_ (given _there are reasons for those feelings_).

So if you expect me to expose my cheeks, interchangeably, for being slapped, in response to  your "if I was from the nation, I would"..., I'm gonna disappoint you as you know: I'm not Jesus and hence lack His humility, as well as I have my reasons to doubt you are the exact Pontius Pilate in order to assign me the cross which _you think_ fits me the best.

**I know where I stand in all this, and how shitty it is for everyone who gives at least a fraction of the whole fuck about it, and it doesn't come for a second without the guilt you mention.**

From the other hand I have my self respect, and no, it's not vanished only because I'm Russian, and it won't. 

It's worth nothing to mention instinctive fight-back reaction. I'm not gonna tolerate systematic harassment way beyond the reasonable iterating thru the arguments why one thinks that Russians are bad enough to have any business with.

That's what I'm doing here and what I'm doing offline, and _to my big luck_ I experience people' understanding.


So:

1. If you'd better see Russians to hold responsible for the atrocities, to act correspondingly, to pay reparations to Ukraine in the end; for some particular human beings - to take their part of responsibility, at moral or criminal plane, depending on the way of them being involved, to process and realise the horrifying truth about them being _related to all of this, regardless_: you need to pass you reality checks better as you want it or not, by "all Russians are the same" approach

**you equate random Russians and ones raping and torturing 8-12 year girls in dark cold basements and those shooting peaceful citizens point blank leaving them _rot on the streets like they are carrion, pieces of flesh_, fatally interrupting their bike ride or walking home to their families from some shopping**

After I wrote this: if you still struggle to see **where the problem is**: I believe you'd better get some help somewhere, as I'm afraid you have actual serious troubles. I won't be able to provide this help as first - you never would accept it and also because my effectiveness would be equal to hitting my head against the wall.

_Zero attempt to assault you as a person_, man! 

But I'm out of words. You may want to ask some person you trust, not me, and with quite high probability you'll get a hint which is not too much different from mine.


2. It's another story if you are _not_ aiming Russians taking their responsibility and _acting_ correspondingly, but are rather driven by vengeance and violence, than it's completely different. If you are to choose the path of war it evades any chance to have _any_ common grounds whatsoever, even in terms of minimum viable discussion. Even in this case, I can understand your choice, but that's my response to it. I'm not evil person, I don't have any hate or so, seriously and honestly, I just try to deduct what I can out of what I have. 

It's very sad for me, that you are reasonable and good person from what I can see, and at the same time -  it's very challenging to keep the dialog without anyone of us sliding to accusations, sticking the labels, etc. That's sort of shitty and when I acknowledge doing so, I reflect and become ashamed of.

> 
> I want to meet no Russians face-to-face, it'd be bad for them. I mean I would not hurt them physically (I would be ashamed to touch them even), but Id tell them real truth about all this.

I'm fine if you meet me one day and tell me the truth in my face, given you'd stick to your avoiding physical violence which I wouldn't tolerate.

I believe in my principles and try my best to align my behaviour with them. Of course, these principles may vary between us, but _I can respond to what is "incriminated"_ to me, that is - being consistent in my line of thoughts. And have my conscience clear which is something which matters for me.

It could've very well been that I'd need to shut up in some particular moments indeed. But my ass normally doesn't go off when I sometimes acknowledge my being wrong, and need to accept the opponents' point of view as a result.

**Comment by ink-splatters** [comment]
> > puppet "liberal" cuckolds in todays Ru politics spewing nonsense and doing nothing to stop the war
> 
> Это кто, например?

Давай на языке треда?

- Venedictov
- Sobchak

ones been to Bolotnaya but manifested their true selves later, regarding the persons mentioned above: look for the Venedictov's statements on the annexation of Crimea.

As for Sobchak, she seems [to have farted in a puddle so remarkably loud](https://youtu.be/0Me_1nJipfA?t=37) that all the attempts to keep good face broke to pieces, effectively exposing the stinky internals.

So that, there are no single doubt about her affiliation with the criminal regime.

**Comment by wojtekxtx** [comment]
> What exactly has made you feel offended?

Seriously? By doing what you Russians know best: by throwing **baseless** allegations.

> not only was it some 10 years ago,

Doesnt matter one bit; could have been 10 years ago, could as well be 100 years ago: IT DOES NOT MATTER. What matters is that she proved to herself (and whole world) that she is perfectly sane. Noone wants to create srious relation with Russian person, honestly I understand her fully. From the time's perspecive she was right. 10 years on your country attacked (lets not fear this word) her country. For the reason of? Someone's fucked up ideas. See the craziness?

> In the most of the criminal laws all over the world this is probaby more known as (in)sanity and if a person commits a crime being in a frenzy, they cannot be appointed legally responsible for their actions and instead of that are placed in mental ward or discharged whatsoever.

True, but one simple thing: crimes against humanity (war crimes are in against humanity) are exception from this rule. Meaning you cannot justify war crimes with insanity. Thats the thing.

> If you are to choose the path of war it evades any chance to have any common grounds whatsoever,

Exactly way round. Its RF that chose path of war, chances were evaded by Putin. So, dont point these words at me!

> I'm afraid you have actual serious troubles

What the f*ck?? The one having trouble

* understanding all this,
* being arrogant,
* offending people,
* accepting full responsibility as any sane person should

is YOU. As you are Russian, that shouldnt be suprising. Your mentality runs deep.

**Comment by schernichkin** [comment]
> @schernichkin and look who is here...... (yet) another Russian citizen. What the hell?

And whom you did expect to see here? This war is our common pain. By "our" I mean Russian and Ukranian sitizens. And yes, if the circumstances are like this, we, probably, having something to discuss among ourselves. How to counter this madness and how the average person could assist to make this stop.

My question, is what are *you* doing here? Poland is not a party of conflict. And your statements are solely about the growth of ethnic hatred. So, I address your own question back to you: what the creeping hell with you?

**Comment by wojtekxtx** [comment]
> My question, is what are you doing here?

@schernichkin compared with you, I have a freedom in deciding where I want to go. 

> How to counter this madness

The easiest and most affordable way is to spread a word = let as many people as possible be against FR and Putin. Thats or start.

> Poland is not a party of conflict

Military not, but dont forget that its Poland that welcomed over 3 milion Ukrainian refugees **for free** so far. Number raising each & everyday.

**Comment by schernichkin** [comment]
@wojtekxtx compared with you, I have responsibility for every single step my government does. Whether I agree with him or not. This is my personal stamp of responsibility: https://imgbb.com/fFK06b4. Every single Russian would be responsible to ukrains. But not to other met propaganda.

**Comment by ink-splatters** [comment]
> > What exactly has made you feel offended?
> 
> Seriously? By doing what you Russians know best: by throwing **baseless** allegations.


```
A.D. 2101
Space mission by Allegation Allies sent to give back the _TRUTH_
which some systems' inhabitants had previously failed to accept, by this having created
Great Singularity Trench at Reasoning dimension

Captain Allegation: What happen?
Mechanic Allegation: Somebody set up us the bomb.
Operator Allegation: We get signal.

Captain Allegation: What!
Mechanic Allegation: Main screen turn on.
Captain  Allegation: It's You!!
Cats: How are you gentlemen!!
Cats: All your base are belong to us. And you BASELESS ALLEGATIONS ! Are on the way to destruction!
Captain: What you say!! WE NEVER HAVE BEEN BASELESS
Cats: Here is your first time suckers. BASELESS AND LIFELESS! Two great ones to happen at once..
Cats: You have no chance to survive make your time. Ha Ha Ha Ha .... 
Operator: Captain!!
Captain: Take off every 'Zig'!!
Captain: You know what you doing.
Operator Allegation: It's all noise! I can't hear you, can you repeat!
Have you said: REWRITE ALL OUR SPACECRAFT FIRMWARE IN ZIG?

Captain Allegation: You said AWARE? Of What? I'm not sure what you are saying, they are
messing up with our radio! Let's manoeuvre like I told you! NOW!!!

Operator Allegation: *rewrites all the systems firmware in Zig programming language*
Captain Allegation: for great justice, man, what the fuck have you done. Are you Sirius? What the crap have you just flashed? Nothing fucking works! 

Operator Allegation: sorry, cap, I forgot to bless the boot image.
Operator Allegation: *blesses boot image* God bless me, otherwise cap would fuck me before throwing my abused body in the outer space!

All the systems boot normally. 

Operator Allegation: but it worked! it worked, and see! they are turning back! ZIG Saved Us All!

Captain Allegation: you just have miraculously saved your asshole's integrity, otherwise would be irreversibly lost and yourself - doomed!


Allegations mission with their base being stronger than ever, continued to its goals...

```

![base](https://user-images.githubusercontent.com/2706884/209012183-58e150c1-bfb4-4a0e-9fc5-7fa80baf0842.png)


> 
> > not only was it some 10 years ago,
> 
> Doesnt matter one bit; could have been 10 years ago, could as well be 100 years ago: IT DOES NOT MATTER. What matters is that she proved to herself (and whole world) that she is perfectly sane. Noone wants to create srious relation with Russian person, honestly I understand her fully. From the time's perspecive she was right. 10 years on your country attacked (lets not fear this word) her country. For the reason of? Someone's fucked up ideas. See the craziness?
> 
> > In the most of the criminal laws all over the world this is probaby more known as (in)sanity and if a person commits a crime being in a frenzy, they cannot be appointed legally responsible for their actions and instead of that are placed in mental ward or discharged whatsoever.
> 
> True, but one simple thing: crimes against humanity (war crimes are in against humanity) are exception from this rule. Meaning you cannot justify war crimes with insanity. Thats the thing.
> 
> > If you are to choose the path of war it evades any chance to have any common grounds whatsoever,
> 
> True.
> 
> > I'm afraid you have actual serious troubles
> 
> What the f*ck?? The one having trouble
> 
> * understanding all this,
> * being arrogant,
> * offending people,
> * accepting full responsibility as any sane person should
> 
> is YOU. As you are Russian, that shouldnt be suprising. Your mentality runs deep.
> 
> > I know where I stand in all this, (...) and it doesn't come for a second without the guilt you mention.
> 
> Now thats what sounds like straightout lie. If this was true, you would:
> 
> * remove yourself from public life ( I mean internet ),
> * not be writing this
> * give back your citizenship of RF
> 
> > According to the psychology basics.......
> 
> ....... when someone commits war crimes, (s)he is INSANE and should be dealt with accordingly. Period.
> 
> > I'm not gonna tolerate systematic harassment
> 
> Now you sound like a f_cking prick. You (personally) are not going to tolerate, but Ukrainians have to? What a f_cked up logic is this?!
> 
> > Zero attempt to assault you as a person, man!
> 
> and
> 
> > you'd better get some help
> 
> What the *? Saying that one should get help is (by every CIVILIZED standard) considered as assault. Not to mention that:
> 
> * it insults me,
> * its highly inappriopriate for you (as a represetant of RF) to say this to anyone.

**Comment by ink-splatters** [comment]
>What matters is that she proved to herself (and whole world) that she is perfectly sane. Noone wants to create srious relation with Russian person, honestly I understand her fully. From the time's perspecive she was right. 

Sorry to say man, but for the sake of the factual correctness I feel urged to disclosed this concrete bit of my private life: it's indeed myself broke with her back then, by reasons completely off our topic, seriously. 

I mean: Siriously

**Comment by ink-splatters** [comment]
>What the f*ck?? The one having trouble
understanding all this,

No, I don't seem to have one.

>being arrogant,
offending people,

I didn't plan to, but it seems I don't have any troubles with offending people and being arrogant either: you feel pretty much offended, don't you? It works! Unfortunately. 

>accepting full responsibility as any sane person should

I thought I've been doing exactly this since the Putler's bloody invasion emerged.

Then I learned from you that `All Russians are the same` which amazed and terrified me. 

I experienced mind blowing interference with the still-standing of the founding pillars which my leaking roof fucking holds itself upon.

```
I started to doubt that if we are all the same, it may vary. As you might not have known,
there was NO SEX in USSR, so we got breeded via facility that you Githubbers nowadays call
`CoW` (copy-on-write). 

We just called it _Корова_ (Cyrillic [Koróva]), by the way, back then. But it doesn't
play too big now.

The only important is that an Hour has come to let the truth out of darkness:
``` 

---
```
there is just one single fucking Russian who is left in the whole Universe, with all the
others being weak replicas, by default just serving über Russian's view controllers.
```
---
```
So it's all a bit complicated here and we need to commence the rules for responsibility
sharing given complicated relations between the replicas with their
# root: ~

Obviously this should be `WEB3 distributed consent based project`: we are all the same but
we are not fucking retarded dinosaurs! 
```

**Comment by wojtekxtx** [comment]
> https://imgbb.com/fFK06b4

@schernichkin whats this? I dont read/speak Russian.

**Comment by Cholowao** [comment]
Defending Innocent Ukrainian citizen and then banning innocent Russian citizens after failing to ban Russian army.. 🙄

**Comment by wojtekxtx** [comment]
@Cholowao what?

**Comment by xamidi** [comment]
I find it amazing how much diplomacy some people use in order to call others out on their xenophobic and/or racist statements.
But not in a good way, since calling it only a "dick stance" is quite an understatement.

Apparently, several individuals here are lacking basic ethical understandings, so I think that unfortunately the following needs to be said.

- Judging individuals not based on their behavior but on traits that they didn't decide for themselves (like skin color or place of birth), is unethical and baseless.
- Individuals cannot be blamed for the actions of another individual only because they share ethnic properties.
- Being Russian is not reprehensible. Supporting Putin (or literally any other murderer) is.
- Dropping statements like "The fact he lives outside of Russia doesnt change the fact he is still russian" — while implying that companies supporting them per se shouldn't be supported — is racist and/or xenophobic.
- The fact that the government of some country is a bunch of violent sociopaths doesn't make you less of a racist and/or xenophobe for discriminating against people from that country in general.
- Many Russians didn't choose war and are in fact against invading other countries and against murder (like all sane individuals), but they had no power to avoid this. The issue in general is the actions of a few individuals.
- It is a common problem in our systems with social hierarchies that powerful individuals do reprehensible things that the powerless masses cannot avoid. Regardless of illusions being created by elections and such.

I didn't think I'd stumble across such a despicable display of terrible education on a network about software development.
All it takes is to combine a little unbiased observation with some logical thinking, and the above statements become rather obvious.

Notably, your collectivist way of thinking is common with individuals that have been mentally crippled by governments via their school systems in order to be submissive and put social conventions above rhyme and reason. Which, ironically, is how wars can happen at all, instead of being shut down by people enforcing reasonable self-justice against the actual perpetrators, like Putin. Because those few who'd have the opportunity to change things for the better usually lie in mental chains.

**Comment by webknjaz** [comment]
> Apparently, several individuals here are lacking basic ethical understandings, so I think that unfortunately the following needs to be said.

Yet, your post is highly judgmental. I don't think it's acceptable for somebody whose culture hasn't been exterminated for hundreds of years to criticize the perception of people being terrorized repeatedly.

>     * Judging individuals not based on their behavior but on traits that they didn't decide for themselves (like skin color or place of birth), is unethical and baseless.

Yes. This is why there's a label that serves as a signal for such *behavior*.

>     * Individuals cannot be blamed for the actions of another individual only because they share ethnic properties.

Yes, the behavior is one of the contributing factors. Like tolerating the terrorists.

>     * Being Russian is not reprehensible. Supporting Putin (or literally any other murderer) is.

There's a lot of factual correlation between being a r*ssian and a ruscist. It's not always the case but the behavior reveals it most of the time.

@xamidi this specific statement of yours perfectly aligns with the excuses of the r*ssian “liberals” for supporting the genocide — dumping the responsibility on the opposite leader. Just remember how the journalists couldn't find any Nazi after the WWII. Everybody said they've never supported Hitler. Well, the historians have a term for describing such people that did nothing. The term is Nazis. Another term you can find used by historians is “Good Germans”.

Now, there's a related turn — ruscists. It doesn't necessarily imply the nationality or where one lives. It's mostly about the social stance that is visible in one's actions or lack of them.

>     * Dropping statements like "The fact he lives outside of Russia doesnt change the fact he is still russian" — while implying that companies supporting them per se shouldn't be supported — is racist and/or xenophobic.

The statement is true. Your implications about what one puts behind the statement are again aligned with the genocide apologists' ones. The passport color is indeed one of the best measurable correlations. Companies injecting r*ssians into workplaces with Ukrainians working there is insensitive at best. But really this contributes to the terror that the ruscists do.

>     * The fact that the government of some country is a bunch of violent sociopaths doesn't make you less of a racist and/or xenophobe for discriminating against people from that country in general.

See above. Nazis never existed, Hitler personally murdered millions of human beings, right?

>     * Many Russians didn't choose war and are in fact against invading other countries and against murder (like all sane individuals), but they had no power to avoid this. The issue in general is the actions of a few individuals.

Basically, this is the same whitewashing of the ruscists as above.

>     * It is a common problem in our systems with social hierarchies that powerful individuals do reprehensible things that the powerless masses cannot avoid. Regardless of illusions being created by elections and such.

This isn't an excuse for behaving like monsters, is it?

> I didn't think I'd stumble across such a despicable display of terrible education on a network about software development. All it takes is to combine a little unbiased observation with some logical thinking, and the above statements become rather obvious.

Except most of the statements are highly biased and illogical.

> Notably, your collectivist way of thinking is common with individuals that have been mentally crippled by governments via their school systems in order to be submissive and put social conventions above rhyme and reason. Which, ironically, is how wars can happen at all, instead of being shut down by people enforcing reasonable self-justice against the actual perpetrators, like Putin. Because those few who'd have the opportunity to change things for the better usually lie in mental chains.

How did you come up with the mental gymnastics of repeating the r*ssian propaganda implying that Ukrainians are brainwashed just because we actually study history?

**Comment by webknjaz** [comment]
@xamidi I won't be responding to your accusations since you've demonstrated how you disregard anything I say, inventing fictional "facts" about me in particular and employing term substitution. I've seen enough and you're mostly trolling or making excuses for the genocide apologists, being insensitive in general, coming to this place. Supporting the ruscist terrorism is not the reason this repository was created, so I'm closing this circus.
